doc/nb-share/notebook-sharing.md

```diff
@@ -1,18 +1,13 @@
 # Sharing notebooks
 
-```{note}
-If you're using JupyterLab 4, you might encounter issues. If the link isn't visible after confirming upload, open the [platform](https://www.platform.ploomber.io/notebooks) to get the link to your notebook.
-```
-
-
 You can use Ploomber Cloud to quickly share notebooks from JupyterLab with one click.
 
 ![](../static/jupyterlab/share-notebook-demo.gif)
 
 First, install the JupyterLab extension:
 
 ```sh
-pip install jupysql-plugin
+pip install ploomber-extension
 ```
```

Comment:
we can remove this part entirelt, I made ploomber-extension a dependency of jupysql

Reply:
oh, my bad. I thought this was jupysql docs!